### ISSUE TITLE
With the old tabbar, it's not possible to have a lower opacity than the text content

### ISSUE THREAD

**Issue by csdvrx** [wez/wezterm#3563]
### What Operating System(s) are you seeing this problem on?

Linux Wayland

### Which Wayland compositor or X11 Window manager(s) are you using?

hyprland

### WezTerm version

20230408-112425-69ae8472

### Did you try the latest nightly build to see if the issue is better (or worse!) than your current version?

No, and I'll explain why below

### Describe the bug

When using the old tabbar with:

    text_background_opacity = 0.4,

    window_background_opacity = 0.7,

I'd expect the text background opacity to add up to the window background opacity

However, the tab bar is still mostly translucent. Inverting the 2 numbers only makes both about the same translunancy

I'd like to have a fully opaque old tabbar, with a #000000 black background, while the textual window content would be mostly transparent.

It seems that would require an extra option to be able to add more opacity just to the tabbar, like: `tabbar_background_opacity=1`

If I'm not correctly using the options and misunderstanding how to configure opacity, I'd love to know how to do that right!

### Configuration

 font = wezterm.font_with_fallback {'Ligconsolata','Deja Vu Sans Mono','HanaMinB','HanaMinA', 'FontAwesome',},
 font_size = 18,
 xcursor_theme = 'Adwaita',
 window_close_confirmation = "NeverPrompt",
 check_for_updates = false,
 enable_wayland = true,
 window_background_opacity = 0.4,
 -- window_background_tint = 0.2,
 text_background_opacity = 0.7,
 inactive_pane_hsb = {
  saturation = 0.9,
  brightness = 0.8,
 },
 enable_tab_bar = true,
 use_fancy_tab_bar = false,
 hide_tab_bar_if_only_one_tab = true,
 colors = {
  tab_bar = {
   active_tab = {
     fg_color = '#d0d0d0',
     bg_color = 'rgba(0,0,0,1)',
     intensity = 'Bold',
     underline = 'Single',
   },
   inactive_tab = {
     fg_color = '#808080',
     bg_color = 'rgba(0,0,0,0.9)',
     intensity = 'Bold',
     underline = 'Single',
   },
   -- old tab bar
   background = "rgba(0,0,0,1)",
   inactive_tab_edge = '#000000',
  },
  -- text override: to grey
  foreground            = "#FDF6E3",
  background            = "#000000",
  split                 = "#444444", -- The color of the split lines between panes
 },
 -- minimal padding is left,right=2
 window_padding = {
  left = 0,
  right = 0,
  top = 0,
  bottom = 0,
 },

**Comment by wez** [comment]
Correct; the retro tab bar is implemented simply as a row of regular text at the top of the screen.
It doesn't composite over the terminal display.

**Comment by csdvrx** [comment]
Would it be then possible to add an opacity setting specific to this row of regular text?

It should have little impact on anything else, while allowing the tab bar to be more opaque, which can be important when not using decorations (like if there's no titlebar)

**Comment by wez** [comment]
Possible? Probably, but it is not necessarily simple. The fancy tab bar is the current solution for more elaborate presentation requirements.

**Comment by csdvrx** [comment]
> Possible? Probably, but it is not necessarily simple. 

Too bad, I'd love the option!

> The fancy tab bar is the current solution for more elaborate presentation requirements.

If I use enable_fancy_bar=true then the background of each tab button is black, but past the last tab button it becomes grey.

Also, the buttons themselves are surrounded by a thin grey border on the left and on the top. 

Changing background = doesn't help.

So may I please ask how I could make everything black in the new titlebar?
